build.sbt

```diff
@@ -21,6 +21,13 @@ asciiGraphWidth := 999999999 // to ensure Snyk can read the the deeeeep dependen
 libraryDependencies ++= Seq(
   "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
   "com.gu.play-googleauth" %% "play-v30" % "19.0.0",
+  "com.google.auth" % "google-auth-library-oauth2-http" % "1.33.1",
+  "com.google.cloud" % "google-cloud-bigquery" % "2.43.1",
```

Comment:
I think `google-cloud-bigquery` is the only dependency you needed to add here. You can confirm this by removing the others and recompiling